README.md

```diff
@@ -65,9 +65,9 @@ Create an **.env** file on root directory
 ```.sh
 SKIP_PREFLIGHT_CHECK=true
 
-REACT_APP_KEYCLOAK_URL=http://sso.sumelms.sol
-REACT_APP_KEYCLOAK_REALM=sume
-REACT_APP_KEYCLOAK_CLIENT_ID=sume-app
+VITE_APP_KEYCLOAK_URL=http://sso.sumelms.sol
+VITE_APP_KEYCLOAK_REALM=sume
+VITE_APP_KEYCLOAK_CLIENT_ID=sume-app
```

Comment:
can you include the mock env here? `VITE_APP_BASE_SERVER_URL=http://localhost:9000/api`